README.md

```diff
@@ -15,6 +15,25 @@ This version will look for an `app_config.yml` file on `config` folder in the pr
 - Add option for server config, right now only creates nginx vhost file and mysql database files for rails AR projects.
 - More stuff
 
+# Use
+In a rails app, simply include the gem in your Gemfile:
+```
+gem 'biran'
+```
+
+In a non-rails app, you will need to do some extra work to make the tasks available to rake. In your Rakefile you will need to manually include things.
+Here is a minimal Rakefile for a basic ruby app that includes the biran tasks along with any local tasks in `lib/tasks`
+```
+require 'bundler/setup'
+require 'biran'
+
+spec = Gem::Specification.find_by_name 'biran'
+load "#{spec.gem_dir}/lib/tasks/biran_tasks.rake"
```

Comment:
Is there an easy way here that I am missing to do a glob on the tasks in that location vs having to load each one? I tried using a wildcard, but Rake wasn't happy
`load "#{spec.gem_dir}/lib/tasks/*.rake"` - no bueno
Would you need to do some sort of each on that dir and loop or can we do something like create a task file to include all the others. Just wondering in case we add other task files later for some reason

Reply:
```
Dir["#{config.gem_dir}/lib/tasks/*.rake"].each do |file|
  Rake::load_rakefile(file)
end
```

Reply:
this is what I found working